moduleframework/common.py

```diff
@@ -540,7 +551,8 @@ def getModulemdYamlconfig(self, urllink=None):
             self.modulemdConf = link
         return link
 
-    def getIPaddr(self):
+    @property
+    def ipaddr(self):
```

Comment:
You are probably changing your public API here, is that correct?

Reply:
Yes, Previously, code looks like `self.getIPaddr()` which returns proper IP and now is easy to use `self.ipaddr`.

@jscotka Your opinion is also welcome. 

Does it make sense?

Reply:
My point is: if anyone is using `self.getIPaddr()` in their tests, this change will break that test and people will need to fix their tests, are you okay with that?

Reply:
@phracek exactly, we have to be backward compatible.
so yep, do that via these getter/setter but have there these old methods too (what will be just wrappers to these new getters/setters)

Reply:
usually projects utilize [`warnings`](https://docs.python.org/2/library/warnings.html) module for such changes and tell people that such functionality is deprecated and will be removed in release X